README.md

```diff
@@ -4,19 +4,19 @@
 
 This repository is used to provide SHACL shapes to verify Health-RI and FAIR Data Point RDF files.
 
-The validator itself is available at <https://www.itb.ec.europa.eu/shacl/healthri/upload>.
+The validator itself is available at <https://www.itb.ec.europa.eu/SHACL/healthri/upload>.
```

Comment:
The uppercase link seems to not redirect properly to the lowercase version. I'd suggest to revert back to the lowercase one.

```suggestion
The validator itself is available at <https://www.itb.ec.europa.eu/shacl/healthri/upload>.
```

Reply:
Yes you're right... i was a overactive when replacing 'shacl' by 'SHACL' as mark asked...